athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/optimizations/pushdown/FilterPushdownSubType.java

```diff
@@ -1,36 +1,42 @@
-package com.amazonaws.athena.connector.lambda.examples;
-
 /*-
  * #%L
  * Amazon Athena Query Federation SDK
  * %%
- * Copyright (C) 2019 Amazon Web Services
+ * Copyright (C) 2019 - 2022 Amazon Web Services
  * %%
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
+ * 
  *      http://www.apache.org/licenses/LICENSE-2.0
- *
+ * 
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  * #L%
  */
+package com.amazonaws.athena.connector.lambda.metadata.optimizations.pushdown;
 
-import com.amazonaws.athena.connector.lambda.handlers.CompositeHandler;
```

Comment:
Yeah I don't know why it's showing the diff this way, but I deleted all the Example* classes that were in this package because they were duplicated (and out of date) from the ones in athena-example. This was in commit `a141f5a2c3f68ee1592582734c8b67337b70b759`